### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by UziTech**
from https://github.com/joyent/node-asn1/pull/13

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
LGTM

File: lib/ber/writer.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ Writer.prototype.writeBuffer = function (buf, tag) {
 
 
 Writer.prototype.writeStringArray = function (strings) {
-  if ((!strings instanceof Array))
+  if (!(strings instanceof Array))
```
Comment (jsumners) [inline lib/ber/writer.js]: I wonder if this should be updated to:

```suggestion
  if (Array.isArray(strings) === false)
```

**Comment by UziTech** [comment]
I changed it to use `Array.isArray`

**Comment by jsumners** [comment]
You should be able to merge this.

**Pull request merged**